db/external_sst_file_test.cc

```diff
@@ -2063,6 +2080,89 @@ TEST_F(ExternalSSTFileTest, LinkExternalSst) {
   }
 }
 
+/*
+ * Test and verify the functionality of ingestion_options.move_files and ingestion_options.failed_move_fall_back_to_copy
```

Comment:
Make format to reduce length of the line.

Reply:
Thanks. I formatted the code.